Gtk-4.0.gir

```diff
@@ -7288,7 +7288,7 @@ See gtk_assistant_commit() for details.</doc>
  add its own buttons through gtk_assistant_add_action_widget().</doc>
       </member>
     </enumeration>
-    <constant name="BINARY_AGE" value="1504" c:type="GTK_BINARY_AGE">
+    <constant name="BINARY_AGE" value="1700" c:type="GTK_BINARY_AGE">
```

Comment:
This is now using GTK 4.17 :melting_face: We should probably first do an import of the final 4.16 (GNOME 47) gir files before updating to the next.

@bilelmoussaoui How can that be done? Just changing to `org.gnome.Sdk//47` or so? And the GNOME 47 SDK gir files are OK now, right?

Reply:
I'd like to do bugfix releases with 4.16 until 4.17 has stabilized a bit at least

Reply:
seems all is good, so merging this one